soc/sensry/ganymed/sy1xx/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2024 sensry.io
+
+zephyr_include_directories(common)
+
+zephyr_compile_options(-march=rv32imc_zicsr -mabi=ilp32)
```

Comment:
`cmake/compiler/gcc/target_riscv.cmake` would generate the correct `march` argument if you set `CONFIG_RISCV_ISA_EXT_*` correctly. So there is no need to specify `march` here.

Reply:
@dcpleung just added PR here: https://github.com/zephyrproject-rtos/zephyr/pull/78633

Reply:
this is merged, would it be ok to "Resolve Conversation" ?